.gitignore

```diff
@@ -352,4 +352,6 @@ MigrationBackup/
 # Ignore sonarqube files
 .sonarqube/
 
-*.idea*
\ No newline at end of file
+*.idea*
```

Comment:
We should keep the `.idea` folder for shared config purposes, they contain their own `.gitignore` files to filter out any unneeded stuff :)

Reply:
`.idea` was already ignored, but happy to ignore it here if we want to keep it!